bedrock/firefox/templates/firefox/all-unified.html

```diff
@@ -26,7 +26,7 @@
 {% endblock %}
 
 {% block site_header %}
-  {% with hide_nav_download_button=True %}
+  {% with hide_nav_download_button=True, hide_nav_get_account_button=True %}
```

Comment:
Here we probably want to hide the download button, but show the accounts button.